### DISCUSSION TITLE
Is is possible to use OASIS3 data with MONAI?

### DISCUSSION THREAD

**Discussion by codingismycraft** [Project-MONAI/MONAILabel#1163] [General]
Reading the documentation it is not clear to me if we can use MONAI with the OASIS3 dataset; is this something that is supported "out of the box"? If not, how difficult will be to add this feature to the existing code?

**Comment by tangy5** [comment]
HI @codingismycraft , thanks for the question. MONAI or MONAI Label are platforms or frameworks. It should be flexible to extend to the OASIS data, we have some preliminary studies using MONAI framework. 

For instance, this application is trained using T1w MRI data including OASIS dataset. 
https://github.com/Project-MONAI/model-zoo/tree/dev/models/wholeBrainSeg_Large_UNEST_segmentation

For MONAI Label, the bundle can be loaded into 3D Slicer and inference registered T1w MRI in MNI305 space like this:
![3DSlicer_use](https://user-images.githubusercontent.com/58751975/204856468-e2d9ac4f-493b-4996-9cf7-4ee8888c8b72.png)



MONAI and Label should have API s to load these MRI data, transforms, and networks that are compatible.

**Reply by jpazarzis** [reply]
@tangy5
many thanks for the quick response.
